app/store/services/geolocation.js

```diff
@@ -0,0 +1,14 @@
+import { getCurrentLocation } from "./helpers/geolocation";
+
+export const getCurrentAthleteLocation = () => {
+  const geolocationOptions = {
+    enableHighAccuracy: true,
+    timeout: 20000,
+    maximumAge: 1000
```

Comment:
i think that could be higher
it depend on the precision you need

we could consider it later